src/VerticalInterpPdf.cc

```diff
@@ -266,12 +266,11 @@ Double_t VerticalInterpPdf::analyticalIntegralWN(Int_t code, const RooArgSet* no
   
   Double_t normVal(1) ;
   if (normSet2) {
-    normVal = 0 ;
-
     TIterator* funcNormIter = cache->_funcNormList.createIterator() ;
 
     RooAbsReal* funcNorm = (RooAbsReal*) funcNormIter->Next();
     central = funcNorm->getVal(normSet2) ;
+    normVal = central;
```

Comment:
Changes here are actually bugfix: Vertical interpolator cannot handle projections in the current code